test/Analysis/Inputs/xtu-other.cpp

```diff
@@ -75,3 +75,10 @@ typedef struct AVBuffer avt;
 int avtSize(void){
 	return sizeof(avt);
 }
+
+template <typename T> class X { T t; };
+
+int fun_with_unsupported_node(){
```

Comment:
Could you reformat this file to match LLVM style? (E.g.: space before the curly. )